octree/include/pcl/octree/octree_base.h

```diff
@@ -102,6 +103,11 @@ namespace pcl
         BreadthFirstIterator breadth_begin(unsigned int max_depth_arg = 0) {return BreadthFirstIterator(this, max_depth_arg);};
         const BreadthFirstIterator breadth_end() {return BreadthFirstIterator();};
 
+        // Octree breadth iterators at a given depth
+        typedef OctreeBreadthFirstAtDepthIterator<OctreeT> BreadthFirstAtDepthIterator;
+        typedef const OctreeBreadthFirstAtDepthIterator<OctreeT> ConstBreadthFirstAtDepthIterator;
+        BreadthFirstAtDepthIterator breadth_at_depth_begin(unsigned int depth_arg = 0) {return BreadthFirstAtDepthIterator(this, depth_arg);};
```

Comment:
Missing space after the function name(s). Missing spaces after `{` and before `}`.